package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "astro-typedoc",
+  "version": "0.1.0",
+  "description": "",
+  "keywords": [],
+  "author": "Felix Khafizov",
```

Comment:
Add your email here. Sometimes people need to contact you on urgent security cases.